octokit/result.go

```diff
@@ -36,6 +45,40 @@ func (r *Result) Error() string {
 	return ""
 }
 
+
+func (r *Result) RateLimitRemaining() int {
+	rate, err := strconv.Atoi(r.Response.Header.Get(rateLimitRemaining))
+	if err != nil {
+		rate = 60
```

Comment:
I don't remember anymore how that thing works :laughing: I guess I can check the response host and set the default to 60 in the case of api.github.com